lnrpc/signrpc/signer.proto

```diff
@@ -220,10 +221,17 @@ message SharedKeyRequest {
     bytes ephemeral_pubkey = 1;
 
     /*
-    The optional key locator of the local key that should be used. If this
-    parameter is not set then the node's identity private key will be used.
+    Deprecated. The optional key locator of the local key that should be used.
+    If this parameter is not set then the node's identity private key will be
+    used.
     */
-    KeyLocator key_loc = 2;
+    KeyLocator key_loc = 2 [deprecated = true];
```

Comment:
This is marked deprecated but is not used at all anymore in the code below. Deprecation means it's still working but people should start using the alternative.

Reply:
fixed